modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
+set of CPUs. This is useful for resource-constrained environments, such as single-node production deployments,
+where you want to reserve most of the CPU resources for user workloads and configure OpenShift to run on a fixed
```

Comment:
Throughout the entire PR, replace "OpenShift" with "{product-title}".

Reply:
Comment accepted.